cmd/cli/exec/exec.go

```diff
@@ -222,6 +222,11 @@ func PrepareJob(cmd *cobra.Command, cmdArgs []string, unknownArgs []string, opti
 			Type:   publisherSpec.Type.String(),
 			Params: publisherSpec.Params,
 		}
+	} else {
+		job.Tasks[0].Publisher = &models.SpecConfig{
+			Type:   "ipfs",
+			Params: map[string]interface{}{},
```

Comment:
why default to ipfs instead of local publisher?

Reply:
Because the local publisher is also currently borked and in need of testing in the RC during the 1.3 release.  So I wanted to play it safe and use IPFS which we know works (mostly).

Reply:
Going to take another look because we shouldn't _need_ to have a default publisher.  I suspect we're getting some defined outputs from somewhere (that requires a publisher) that might be unnecessary.

Reply:
redone.